scripts/kernel-modules-unload.sh

```diff
@@ -23,4 +23,5 @@ fi
 set -o errexit
 set -o xtrace
 
+systemctl stop zfs-zed.service
```

Comment:
why do we need this?

Reply:
I explained in the last sentence of the description; in order to unload the zfs kernel modules, the zed service must be stopped first. If it isn't stopped, modprobe -r fails with EBUSY. It wasn't an issue before, because the utils were only installed inside the container, but now that they're installed in the bootstrap VM itself we need to handle this.

Reply:
I guess I was more curious why we couldn't unload the kernel module without stopping this service.. like, what does this do to prevent EBUSY? do we know?